local-cli/runAndroid/runAndroid.js

```diff
@@ -291,7 +293,7 @@ module.exports = {
     command: '--install-debug',
   }, {
     command: '--root [string]',
-    description: 'Override the root directory for the android build (which contains the android directory)',
+    description: 'Path relative to project root directory where the Android project lives. The default is \'android\'.',
     default: '',
```

Comment:
Just curious, if the default value is `android` why do not setup it here instead of having same custom logic in two places?